tests/unit/s2n_choose_supported_group_test.c

```diff
@@ -30,12 +31,15 @@ int main(int argc, char **argv)
         /* If the list of mutually supported groups is empty, chosen curve should be set to null */
         EXPECT_NOT_NULL(server_conn = s2n_connection_new(S2N_SERVER));
         EXPECT_NULL(server_conn->secure.server_ecc_evp_params.negotiated_curve);
-        for (int i = 0; i < S2N_ECC_EVP_SUPPORTED_CURVES_COUNT; i++) {
+        notnull_check(server_conn->config);
```

Comment:
Is there a reason why you're using notnull_check() versus EXPECT_NOT_NULL()?

Reply:
No particular reason they do the same thing